apps/docs/src/getting-started/cms/content-pages.md

```diff
@@ -48,22 +48,22 @@ export default defineNuxtConfig({
 
 ### Render the page
 
-Now, you can import all components from the `@shopware-pwa/cms-base` package and use them in your templates. The most straightforward way to render a page is to use the `CmsPage` component. It takes a `content` prop and resolves all subordinate sections, blocks and elements automatically. Put the following code in your catch-all component that also handles the [routing logic](./routing.md#resolve-a-route-to-a-page).
+Now, you can import all components from the `@shopware-pwa/cms-base` package and use them in your templates. The most straightforward way to render a page is to use the `CmsPage` component. It takes a `content` prop and resolves all subordinate sections, blocks and elements automatically. Put the following code in your catch-all component that also handles the [routing logic](./../routing.md#resolve-a-route-to-a-page).
 
 ```vue-html
 <CmsPage v-if="data" :content="data.value.cmsPage"/>
 ```
 
 :::warning This will only work
-if you followed the previous chapter on [routing](./routing.md). The `data` value in this example is a reactive reference to either a product, a category or a landing page response. If you are not sure how to get the data otherwise, check that chapter and see how the data is fetched.
+if you followed the previous chapter on [routing](./../routing.md). The `data` value in this example is a reactive reference to either a product, a category or a landing page response. If you are not sure how to get the data otherwise, check that chapter and see how the data is fetched.
```

Comment:
```suggestion
if you followed the previous chapter on [routing](../routing.md). The `data` value in this example is a reactive reference to either a product, a category or a landing page response. If you are not sure how to get the data otherwise, check that chapter and see how the data is fetched.
```